python/hsfs/feature_group.py

```diff
@@ -54,7 +54,7 @@ def __init__(
         hudi_enabled=False,
         statistics_config=None,
     ):
-        feature_group_base.FeatureGroupBase.__init__(featurestore_id)
+        feature_group_base.FeatureGroupBase.__init__(self, featurestore_id)
```

Comment:
here you would use `super().__init__()` since you want to do additional stuff to the initialization of the base class